<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="image">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>500</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Image path is displayed here.</string>
        </property>
        <property name="text">
         <string>Select images using File &gt; Open!</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="vertical_spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="previous_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Previous image.</string>
          </property>
          <property name="statusTip">
           <string>Previous image.</string>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="image_num_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Image 0 of 0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Next image.</string>
          </property>
          <property name="statusTip">
           <string>Next image.</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="apply_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Compute circumference using current settings.</string>
        </property>
        <property name="statusTip">
         <string>Compute circumference using current settings.</string>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="x_rotation_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>X rotation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="x_slider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Control X rotation value. Left click on the left end or right end to decrease or increase angle by 1 degree.</string>
        </property>
        <property name="statusTip">
         <string>Control X rotation value. Left click on the left end or right end to decrease or increase angle by 1 degree.</string>
        </property>
        <property name="minimum">
         <number>-90</number>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="y_rotation_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Y rotation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="y_slider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Control Y rotation value. Left click on the left end or right end to decrease or increase angle by 1 degree.</string>
        </property>
        <property name="statusTip">
         <string>Control Y rotation value. Left click on the left end or right end to decrease or increase angle by 1 degree.</string>
        </property>
        <property name="minimum">
         <number>-90</number>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="z_rotation_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Z rotation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="z_slider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Control Z rotation value. Left click on the left end or right end to decrease or increase angle by 1 degree.</string>
        </property>
        <property name="statusTip">
         <string>Control Z rotation value. Left click on the left end or right end to decrease or increase angle by 1 degree.</string>
        </property>
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slice_num_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Slice</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="slice_slider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Control slice value. Left click on the left end or right end to decrease or increase by 1.</string>
        </property>
        <property name="statusTip">
         <string>Control slice value. Left click on the left end or right end to decrease or increase by 1.</string>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="value">
         <number>90</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Reset rotation and slice values for the current image.</string>
        </property>
        <property name="statusTip">
         <string>Reset rotation and slice values for the current image.</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>24</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="action_export">
     <property name="toolTip">
      <string extracomment="Export image data as CSV or export the image.">Export image data as CSV or export the image.</string>
     </property>
     <property name="statusTip">
      <string extracomment="Export slice data to CSV or to image format.">Export image data as CSV or export the image.</string>
     </property>
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="action_export_csv"/>
     <addaction name="separator"/>
     <addaction name="action_export_png"/>
     <addaction name="action_export_jpg"/>
     <addaction name="action_export_bmp"/>
     <addaction name="action_export_ppm"/>
     <addaction name="action_export_xbm"/>
     <addaction name="action_export_xpm"/>
    </widget>
    <addaction name="action_open"/>
    <addaction name="action_add_images"/>
    <addaction name="action_remove_image"/>
    <addaction name="action_export"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_github"/>
    <addaction name="action_documentation"/>
   </widget>
   <widget class="QMenu" name="menu_debug">
    <property name="toolTip">
     <string>Tools for debugging.</string>
    </property>
    <property name="statusTip">
     <string>Tools for debugging.</string>
    </property>
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="action_test_show_resource"/>
    <addaction name="action_print_metadata"/>
    <addaction name="action_print_dimensions"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_help"/>
   <addaction name="menu_debug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_documentation">
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="toolTip">
    <string>Open documentation website in web browser.</string>
   </property>
   <property name="statusTip">
    <string>Open documentation website in web browser.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit.</string>
   </property>
   <property name="statusTip">
    <string>Exit.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Select image(s) to be opened.</string>
   </property>
   <property name="statusTip">
    <string>Select image(s) to be opened.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_add_images">
   <property name="text">
    <string>Add Images</string>
   </property>
   <property name="toolTip">
    <string>Add image(s) to the end of the list. No duplicates.</string>
   </property>
   <property name="statusTip">
    <string>Add image(s) to the end of the list. No duplicates.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_remove_image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove Image</string>
   </property>
   <property name="toolTip">
    <string>Remove currently displayed image from the list. Does not delete the file from disk.</string>
   </property>
   <property name="statusTip">
    <string>Remove currently displayed image from the list. Does not delete the file from disk.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_export_png">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PNG</string>
   </property>
   <property name="toolTip">
    <string extracomment="Export slice as PNG.">Export slice as PNG.</string>
   </property>
   <property name="statusTip">
    <string extracomment="Export slice as PNG.">Export slice as PNG.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Ctrl+P</string>
   </property>
  </action>
  <action name="action_export_jpg">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>JPG</string>
   </property>
   <property name="toolTip">
    <string extracomment="Export slice as JPG.">Export slice as JPG.</string>
   </property>
   <property name="statusTip">
    <string extracomment="Export slice as JPG.">Export slice as JPG.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Ctrl+J</string>
   </property>
  </action>
  <action name="action_export_csv">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>CSV</string>
   </property>
   <property name="toolTip">
    <string extracomment="Click Apply to enable this option.">Export image data as CSV.</string>
   </property>
   <property name="statusTip">
    <string extracomment="Click Apply to enable this option.">Export image data as CSV.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Ctrl+C</string>
   </property>
  </action>
  <action name="action_export_bmp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>BMP</string>
   </property>
   <property name="toolTip">
    <string extracomment="Export slice as BMP.">Export slice as BMP.</string>
   </property>
   <property name="statusTip">
    <string extracomment="Export slice as BMP.">Export slice as BMP.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E, Ctrl+B</string>
   </property>
  </action>
  <action name="action_export_ppm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PPM</string>
   </property>
   <property name="toolTip">
    <string extracomment="Export slice as PPM.">Export slice as PPM.</string>
   </property>
   <property name="statusTip">
    <string extracomment="Export slice as PPM.">Export slice as PPM.</string>
   </property>
  </action>
  <action name="action_export_xbm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>XBM</string>
   </property>
   <property name="toolTip">
    <string extracomment="Export slice as XBM.">Export slice as XBM.</string>
   </property>
   <property name="statusTip">
    <string extracomment="Export slice as XBM.">Export slice as XBM.</string>
   </property>
  </action>
  <action name="action_export_xpm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>XPM</string>
   </property>
   <property name="toolTip">
    <string extracomment="Export slice as XPM.">Export slice as XPM.</string>
   </property>
   <property name="statusTip">
    <string extracomment="Export slice as XPM.">Export slice as XPM.</string>
   </property>
  </action>
  <action name="action_github">
   <property name="text">
    <string>GitHub</string>
   </property>
   <property name="toolTip">
    <string>Open GitHub website in web browser.</string>
   </property>
   <property name="statusTip">
    <string>Open GitHub website in web browser.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_test_show_resource">
   <property name="text">
    <string>Test show resource</string>
   </property>
   <property name="toolTip">
    <string>Test displaying a resource (help.svg).</string>
   </property>
   <property name="statusTip">
    <string>Test displaying a resource (help.svg).</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_print_metadata">
   <property name="text">
    <string>Print metadata</string>
   </property>
   <property name="toolTip">
    <string>Print metadata to terminal.</string>
   </property>
   <property name="statusTip">
    <string>Print metadata to terminal.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P, Ctrl+M</string>
   </property>
  </action>
  <action name="action_print_dimensions">
   <property name="text">
    <string>Print dimensions</string>
   </property>
   <property name="toolTip">
    <string>Print dimensions to terminal.</string>
   </property>
   <property name="statusTip">
    <string>Print dimensions to terminal.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P, Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
